portlet/exoadmin/src/main/java/org/exoplatform/organization/webui/component/UIAccountEditInputSet.java

```diff
@@ -147,15 +147,21 @@ public boolean save(OrganizationService service) throws Exception {
         Query query = new Query();
         String email = getUIStringInput("email").getValue();
         query.setEmail(email);
-        if (!oldEmail.equals(email) && service.getUserHandler().findUsersByQuery(query, UserStatus.ANY).getSize() > 0) {
+        if (((oldEmail != null && !oldEmail.equals(email)) || oldEmail == null) && service.getUserHandler().findUsersByQuery(query, UserStatus.ANY).getSize() > 0) {
             // Be sure it keep old value
-            user.setEmail(oldEmail);
-            query.setEmail(oldEmail);
+            if (oldEmail != null) {
+                user.setEmail(oldEmail);
+                query.setEmail(oldEmail);
+            }
             Object[] args = { username };
             uiApp.addMessage(new ApplicationMessage("UIAccountInputSet.msg.email-exist", args, ApplicationMessage.WARNING));
             return false;
         }
 
+        if ( oldEmail == null && !"".equals(email) ){
+            user.setEmail(email);
+        }
```

Comment:
Is this really necessary ? I guess the new email is already set by the call to the method `invokeSetBindingField` (https://github.com/exoplatform/gatein-portal/pull/130/files#diff-026b727166f14cff2ca0e4ecde4aaaeaR134), no ?